src/lib/process_crawl.js

```diff
@@ -1,27 +1,197 @@
 var rc_util = require('rippled-network-crawler/src/lib/utility.js');
+var crawler = require('rippled-network-crawler/src/lib/crawler.js');
```

Comment:
Are the things that the reporter uses from the crawler also used by the crawler?
If so, we could move the functionality into the reporter instead of depending on `rippled-network-crawler`